src/db/migration/migrate20230224-UpdateOnlyOffice-7.2.1.pl

```diff
@@ -0,0 +1,56 @@
+# ***** BEGIN LICENSE BLOCK *****
+# Zimbra Collaboration Suite Server
+# Copyright (C) 2023 Synacor, Inc.
+#
+# This program is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software Foundation,
+# version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+# without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+# You should have received a copy of the GNU General Public License along with this program.
+# If not, see <https://www.gnu.org/licenses/>.
+# ***** END LICENSE BLOCK *****
+#
+use DBI;
+use strict;
+use Migrate;
+
+# Verify Schema Version Number
+Migrate::verifySchemaVersion(117);
+
+addTenantColumnToDocChanges();
+addMultiColumnToTaskResult();
+
+# Update Schema Version Number
+Migrate::updateSchemaVersion(117, 118);
+
+exit(0);
+
+# Function to add 'tenant' column to doc_changes table that is auto updating
+sub addTenantColumnToDocChanges() {
+    my $sql = <<MDM_ADD_COLUMN_EOF;
```

Comment:
MDM is for mobile device management, change `MDM_ADD_COLUMN_EOF` to `OO_ADD_COLUMN_EOF`

Reply:
Renamed.